akka-projection-testkit/src/main/scala/akka/projection/testkit/javadsl/ProjectionTestKit.scala

```diff
@@ -14,11 +14,13 @@ import akka.actor.testkit.typed.TestKitSettings
 import akka.actor.testkit.typed.javadsl.ActorTestKit
 import akka.actor.typed.javadsl.Adapter
 import akka.annotation.ApiMayChange
+import akka.japi.function.Creator
 import akka.projection.Projection
 import akka.projection.ProjectionSettings
 import akka.stream.testkit.TestSubscriber
 import akka.stream.testkit.javadsl.TestSink
 import akka.util.JavaDurationConverters._
+import akka.japi.function.Function
```

Comment:
rename that import `{Function => JFunction`} so that we don't change the public signature if it is accidentally removed (scala.Function)